Algorithm-Chunky-Monkey.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Chunky Monkey
+
 ![](http://i.imgur.com/RdrnNvA.png)
 
-# Explanation
+## Explanation

```

Comment:
The Explanation should appear in chat at the first call. With `h2`, it won't appear.